packages/navigation/src/_internal/shouldDisplayCustomSubHeader.ts

```diff
@@ -1,18 +1,12 @@
-import { resolveOptions } from "./resolveOptions";
-import { MadBaseOptions, MadDescriptorBase, UnresolvedScreenOptions } from "./types";
+import { MadBaseOptions } from "./types";
 
 /**
  * Calculate whether the custom sub-header should be displayed or not
  * @param descriptor descriptor from the descriptors object provided by React Navigation
  * @param unresolvedScreenOptions unresolved screen options provided by React Navigation
```

Comment:
Should `descriptor` and `unresolvedScreenOptions` be replaced by `options` in the JSDoc?

Reply:
Fixed